src/Text.Analyzers/Core/IdentifiersShouldBeSpelledCorrectly.cs

```diff
@@ -1,24 +1,34 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.Linq;
+using System.Reflection;
 using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+using Microsoft.CodeAnalysis.Text;
 
 namespace Text.Analyzers
 {
     /// <summary>
     /// CA1704: Identifiers should be spelled correctly
     /// </summary>
-    public abstract class IdentifiersShouldBeSpelledCorrectlyAnalyzer : DiagnosticAnalyzer
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public class IdentifiersShouldBeSpelledCorrectlyAnalyzer : DiagnosticAnalyzer
```

Comment:
```suggestion
    public sealed class IdentifiersShouldBeSpelledCorrectlyAnalyzer : DiagnosticAnalyzer
```

Reply:
Can I remove the CSharpIdentifiersShouldBeSpelledCorrectly/BasicIdentifiersShouldBeSpelledCorrectly classes then?

Reply:
Either you need C#/VB.NET specific APIs in which case this class should be `abstract` or you can do everything from the language agnostic layer in which case this should be `sealed` and you don't need the sub-classes (you can remove them).

Reply:
Done